### PR TITLE
feat: distinct

### PR DESCRIPTION
**Pull request by timcadman**
## Checklist for developing new functions
- [x] Write clientside function
- [x] Write serverside function
- [x] Evaluate whether additional disclosure checks are required, and if so write them
- [x] Update .circleci/config.yml so that CI installs 'molgenis/tidyverse' from corresponding branch
- [x] Add serverside function to inst/DATASHIELD file
- [x] On clientside package run `devtools::check()`. Review and fix any broken tests. 
- [x] On serverside package run `devtools::check()`. Review and fix any broken tests. 
- [x] Add new clientside tests to 'tests/testthat'
- [x] Add new serverside tests to 'tests/testthat'
- [x] Update clientside README.md to list function as included 
- [x] Update clientside vignette with example(s) of new function use
- [x] Check clientside function documentation is correct and complete
- [x] Check serverside function documentation is correct and complete
- [x] On clientside package run `devtools::check()` and `devtools::build_site()`
- [x] On serverside package run `devtools::check()` and `devtools::build_site()`

## How to test
- [ ] Pull feat/distinct branch from molgenis/dsTidyverseClient
- [ ] Pull feat/distinct branch from molgenis/dsTidyverse
- [ ] Run: `devtools::load_all(path-to-clientside-package)`
- [ ] Start Armadillo on localhost
- [ ] Start xenon profile
- [ ] Install package on server:

```
devtools::build(path-to-serverside-package)
local_url <- "http://localhost:8080"
armadillo.login(local_url)
armadillo.install_packages(path-to-built-serverside-package, profile = "xenon")
```

- [ ] Upload test data:
```
data("mtcars")
armadillo.create_project("tidyverse")
armadillo.upload_table("tidyverse", "data", mtcars)
```

- [ ] Log in and assign data:
```
token <- armadillo.get_token(local_url)
builder <- newDSLoginBuilder()

builder$append(
  url = local_url,
  server = "cohort_1",
  token = token,
  driver = "ArmadilloDriver",
  profile = "xenon")

builder$append(
  url = local_url,
  server = "cohort_2",
  token = token,
  driver = "ArmadilloDriver",
  profile = "xenon")

builder$append(
  url = local_url,
  server = "cohort_3",
  token = token,
  driver = "ArmadilloDriver",
  profile = "xenon")

logindata <- builder$build()
conns <- datashield.login(logins = logindata, assign = FALSE)

datashield.assign(
  symbol = "mtcars",
  value = "tidyverse/data/mtcars",
  conns = conns)
```

- [ ] Run the test lines for `tests/testthat/test-distinct.R` Ignore the setting up a DSLite environment.

In addition:
- [ ] run `devtools::check()
- [ ] read vignette

### CODE REVIEW

File: docs/pkgdown.yml [hunk 1]
```diff
@@ -3,5 +3,9 @@ pkgdown: 2.0.9
 pkgdown_sha: ~
 articles:
   dstidyverse: dstidyverse.html
+<<<<<<< HEAD
```
Comment (marikaris) [inline docs/pkgdown.yml:6]: This is not good

File: vignettes/dstidyverse.Rmd [hunk 2]
```diff
@@ -38,9 +38,12 @@ require(DSI)
 
 ```r
 <<<<<<< HEAD
+<<<<<<< HEAD
```
Comment (marikaris) [inline vignettes/dstidyverse.Rmd:41]: What did you do?!

**Review by marikaris — CHANGES_REQUESTED** [review]
Something went terribly wrong with your merge conflicts

**Comment by marikaris** [comment]
Also getting errors when running the first test:
`[1] "Bad request: No such DataSHIELD 'ASSIGN' method with name: distinctDS"`
I think it's because the missing comma's in `inst/DATASHIELD` in the ds-tidyverse repository

**Comment by timcadman** [comment]
> Also getting errors when running the first test: `[1] "Bad request: No such DataSHIELD 'ASSIGN' method with name: distinctDS"` I think it's because the missing comma's in `inst/DATASHIELD` in the ds-tidyverse repository

I've fixed this in the serverside package

**Review by marikaris — APPROVED** [review]

**Pull request merged**